postgres/lib/PostgresService.js

```diff
@@ -290,11 +290,11 @@ GROUP BY k
       return ret === '?' ? `$${this.values.length}` : ret
     }
 
-    operator(x) {
+    operator(x, i, xpr) {
       if (x === 'regexp') return '~'
-      if (x === '=') return 'is not distinct from'
-      if (x === '!=') return 'is distinct from'
-      else return x
```

Comment:
It could be part of the problem, that we completely override the base class impl, hence loosing the `is null` / `is not null` handling. In particular, the example above: 

```sql
SELECT 1 where 1 = null;
SELECT 1 where 1 != null;
```

... would not resolve to this, as it should:

```sql
SELECT 1 where 1 is null;
SELECT 1 where 1 is not null;
```

... but to this, which might be less efficient:

```sql
SELECT 1 where 1 is not distinct from null;
SELECT 1 where 1 is distinct from null;
```